### PR TITLE
Specify source encoding in build.sbt

### PR DESCRIPTION
**Pull request by igrep**
This is required to build this project on my machine: Windows 10 in Japanese.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -21,4 +21,6 @@ lazy val intellijHaskell = (project in file(".")).
     unmanagedJars in Compile += baseDirectory.value / "idea"
   )
 
-ideaBuild in ThisBuild := "163.7743.44"
\ No newline at end of file
+ideaBuild in ThisBuild := "163.7743.44"
+
+javacOptions ++= Seq("-encoding", "UTF-8")
```
Comment (rikvdkleij) [inline build.sbt]: Why are you not adding them to javacOptions inside setting?

Comment (igrep) [inline build.sbt]: Yeah, it's already set in my IntelliJ.
Actually, I was stuck by the error caused by the source encoding when `compile` on sbt console.
Do you know how to set it on the console?
Sorry, I'm new to sbt.

Comment (rikvdkleij) [inline build.sbt]: My default encoding is UTF-8 in IntelliJ. I can not reproduce your issue. Does your change in build.sbt works for you?

Comment (rikvdkleij) [inline build.sbt]: Does it work for you if you add them to: `scalacOptions in Global`

Comment (rikvdkleij) [inline build.sbt]: @zjhmale Do you also have this encoding issue?

Comment (zjhmale) [inline build.sbt]: @rikvdkleij Nope, I currently work with OSX 10.10.5 and I got all my encoding in UTF-8.

image

Comment (zjhmale) [inline build.sbt]: @igrep Can you fix this with changing the default charset in Windows Terminal? IIRC the default encoding in Windows Console is not UTF-8.

Comment (igrep) [inline build.sbt]: Adding `%USERPROFILE%/.sbt/0.13/global.sbt` below worked! 😄 
Thank you!

```scala:global.sbt
javacOptions ++= Seq("-encoding", "UTF-8")
```

Comment (zjhmale) [inline build.sbt]: Great! :smile:

Comment (rikvdkleij) [inline build.sbt]: So we can close this PR?

Comment (zjhmale) [inline build.sbt]: LGTM
> So we can close this PR?

Comment (igrep) [inline build.sbt]: Yes. Thanks for the advice!

**Pull request closed without merging**